tethys_apps/templatetags/site_settings.py

```diff
@@ -14,23 +14,27 @@
 @register.filter
 @stringfilter
 def load_custom_css(var):
-    if var.startswith("/"):
-        var = var.lstrip("/")
+    if var.startswith(os.sep):
```

Comment:
I'm not sure that this is the right move here. I'm trying to decide if the user should specify these as a URL path or a system path.